temoa/extensions/template/components/example_limit.py

```
@@ -0,0 +1,62 @@
+
+This example caps the new capacity built in each period for a technology (or
+technology group) in a region. Replace it with your own logic.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from pyomo.environ import Constraint, quicksum, value
+
+import temoa.components.geography as geography
```

Ruff (0.15.20)

[warning] Use `from temoa.components import geography` in lieu of alias

Replace with `from temoa.components import geography`

(PLR0402)